src/libraries/System.Private.Xml/src/System/Xml/Xslt/XsltArgumentList.cs

```diff
@@ -43,6 +44,8 @@ public void AddParam(string name, string namespaceUri, object parameter)
             _parameters.Add(qname, parameter);
         }
 
+        [RequiresUnreferencedCode("The stylesheet may have calls to methods of the extension object passed in which cannot be statically analyzed " +
```

Comment:
Don't forget to update the `ref` code as well.

Reply:
Right 👍 , I'm planning to do that once I'm done with the next set of changes which should be pushed real soon.